src/Middleware/RetryMiddleware.php

```diff
@@ -51,14 +51,23 @@ class RetryMiddleware
     /** @var float|null */
     private $deadlineMs;
 
+    /*
+     * The number of API calls(not retries) that have already been attempted.
+     * The first retry will have $currentAttempts set to 1.
+     * @var int
+     */
+    private $currentAttempts;
```

Comment:
It seems like this should be named `$retryAttempt` instead of `$currentAttempts`? 

see [ExponentialBackoff](https://github.com/googleapis/google-cloud-php/blob/e9ed1a156a32a07bc6231001fa431d1c162f452f/Core/src/ExponentialBackoff.php#L100)

Reply:
Actually this included the original API attempt to reflect the number of total attempts to do the operation. I'll change it back to how it is in the ExponentialBackoff file.